### PR TITLE
Feature/form contact

### PR DESCRIPTION
**Pull request by ravinalamada**
Hi Team,

We've added a new pattern!

**Description**

We were working on the contact form. It is a form that enables a user to interact with the owner of the website. We made it work responsively with mobile phones and desktops. We have uploaded its image below. Please have a look at our code whenever you have a chance.
Thanks in advance.

**Screenshots**
 
_C__Users_Student_code_onja-website_contact_form html
)

 
**Checklist**

- [x] Merged to `testing`
- [ ] Cross-browser tested
- [ ] Files are in the right places
- [ ] Suitable names have been chosen for `class`es, `id`s and files.
- [ ] New patterns have a corresponding HTML file, that has all the relevant test cases within.

### CODE REVIEW

File: css/share.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+fieldset {
+    border: transparent;
+
+}
+
+input, textarea {
+  width: 100%;
+  height: 100%;
+}
```
Comment (Betsimisaraka) [inline css/share.css]: It is nice to see, what you have done but you should use class or Id to explicit your selector.

File: css/share.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  #fieldset-4 {
+    grid-column: 2;
+  }
+
+  #fieldset-5 {
+    grid-column: 1/3;
+
+  }
+ input, textarea {
```
Comment (Betsimisaraka) [inline css/share.css]: This one as well.

File: css/share.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  #fieldset-3 {
+    grid-column: 1;
+  }
+
+  #fieldset-4 {
+    grid-column: 2;
+  }
+
+  #fieldset-5 {
```
Comment (Betsimisaraka) [inline css/share.css]: It better to have a good name of the Id.

File: css/share.css [hunk 4]
```diff
@@ -9,7 +9,7 @@ textarea {
   width: 100%;
 }
 
-@media (min-width: 647px) {
+@media (min-width: 480px) {
     .contact-form {
```
Comment (Betsimisaraka) [inline css/share.css]: Please fix your indenting.

File: contact_form.html [hunk 5]
```diff
@@ -37,6 +37,7 @@ <h2 class="form_contact__heading">
       <button type="submit" name="button-form" class="button button__primary button__primary--button-send" id="submit-button">Send</button>
     </form>
   </section>
+
   <section class="form_contact__section">
     <header class="form_contact__header">
       <h2 class="form_contact__heading">
```
Comment (Betsimisaraka) [inline contact_form.html:43]: This looks great guys, well done.

File: share.html [hunk 6]
```diff
@@ -15,25 +15,25 @@ <h2>
         Contact form
       </h2>
     </header>
-    <form action="/contact-form" method="POST" class="contact-form">
-      <fieldset class="contact-field" id="fieldset-1">
-        <label class="contact-label" for="fullname" id="fullname-label">Fullname</label><br>
-        <input type="text" id="fullname">
+    <form class="contact-form" action="/contact-form" method="POST" id="contact_form">
```
Comment (Betsimisaraka) [inline share.html]: It looks like you are trying to use BEM but it is not really clear.

**Review by Betsimisaraka — COMMENTED** [review]
You have done really good work but there is only little comment.
You should write a good commit message and please check the spelling of your commit message.

File: css/share.css (hunk 1 above)
Comment (ravinalamada) [reply css/share.css]: Thank you for your suggestion

File: css/share.css (hunk 3 above)
Comment (ravinalamada) [reply css/share.css]: Do you mean that it is better to use an ID?

**Comment by ravinalamada** [comment]
Hi team,

We are so happy to receiving your suggestion. 

Thank you for reviewing our code that is very kind of you.

Prisca and Rinon

**Pull request closed without merging**